src/DesignQuery/DeisgnQuery_tcl_commands_for_sdt_generation.tcl

```diff
@@ -0,0 +1,21 @@
+# These commands are in the FOEDAG folder in the file gbox_top.tcl
```

Comment:
The file name needs to be fixed: 
**DeisgnQuery**_tcl_commands_for_sdt_generation.tcl